src/fragments/lib/graphqlapi/flutter/getting-started/10_preReq.mdx

```diff
@@ -1,5 +1,12 @@
 * [Install and configure Amplify CLI](https://docs.amplify.aws/cli/start/install)
-* A Flutter application targeting Flutter SDK >= 2.10.0 (stable version) with Amplify libraries integrated
-  * An iOS configuration targeting at least iOS 11.0
-  * An Android configuration targeting at least Android API level 21 (Android 5.0) or above
+* A Flutter application targeting Flutter SDK >= 3.3.0 with Amplify libraries integrated
+  
+  The following are also required, depending on which platforms you are targeting:
+
+  * An iOS configuration targeting at least iOS 13.0 and XCode version >=13.2
+  * An Android configuration targeting at least Android API level 24 (Android 7.0) or above
+  * Any browser supported by Flutter for Web (you can check the list of supported browsers [here](https://docs.flutter.dev/development/platform-integration/web/faq#which-web-browsers-are-supported-by-flutter))
+  * Any Windows OS meeting Flutter minimums
+  * macOS version 10.15 or higher
```

Comment:
I think we should update this language as the bullet for macOS doesn't really cover all the requirements. I will open a separate PR to do that though since this needs to be updated in several places.